docs/.vitepress/theme/styles/landing.css

```diff
@@ -3,17 +3,15 @@
 /* /////////////////////// */
 
 html:has(.landing) {
+  --vp-c-bg: #101010;
+
   background-color: #101010;
 
   body {
     background-color: #101010;
   }
 }
 
-html.dark:has(.landing) {
-  --vp-c-bg: #101010;
-}
```

Comment:
The landing page is forced to dark mode (see the CSS below), so this `.dark` should not be added.